specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json

```diff
@@ -5,64 +5,15 @@
     "title": "Computer Vision Client",
     "description": "The Computer Vision API provides state-of-the-art algorithms to process images and return information. For example, it can be used to determine if an image contains mature content, or it can be used to find all the faces in an image.  It also has other features like estimating dominant and accent colors, categorizing the content of images, and describing an image with complete English sentences.  Additionally, it can also intelligently generate images thumbnails for displaying large images effectively."
   },
-  "securityDefinitions": {
-    "apim_key": {
-      "type": "apiKey",
-      "name": "Ocp-Apim-Subscription-Key",
-      "in": "header"
-    }
-  },
-  "security": [
-    {
-      "apim_key": []
-    }
-  ],
-  "x-ms-parameterized-host": {
```

Comment:
"x-ms-parameterized-host" is Microsoft's extension for replacing "host", "basePath", "schemes". I think it's better to keep them close to "host"